pkg/util/kubernetes/apiserver/leaderelection/leaderelection.go

```diff
@@ -177,6 +180,30 @@ func (le *LeaderEngine) IsLeader() bool {
 	return le.CurrentLeaderName() == le.HolderIdentity
 }
 
+// GetLeaderDetails is used in for the Flare and for the Status commands.
+func GetLeaderDetails() (leaderDetails rl.LeaderElectionRecord, err error) {
+	var led rl.LeaderElectionRecord
+	c, err := apiserver.GetCoreV1Client()
+	if err != nil {
+		return led, err
+	}
+	leaderElectionCM, err := c.ConfigMaps(metav1.NamespaceDefault).Get(defaultLeaseName, metav1.GetOptions{})
+	if err != nil {
+		return led, err
+	}
+	log.Infof("LeaderElection cm is %q", leaderElectionCM)
+	annotation, found := leaderElectionCM.Annotations[rl.LeaderElectionRecordAnnotationKey]
+	if !found {
+		return led, apiserver.ErrNotFound
+	}
+	bytes := []byte(annotation)
```

Comment:
Careful because bytes is also a package.
You can directly do:
```go
err = json.Unmarshal([]byte(annotation), &led)
```
?

Reply:
yeah I guess I should avoid using package names as variables overall 😅